### PR TITLE
Deepstream gstreamer: Loading detectors from config

### PR DESCRIPTION
**Pull request by JsonSadler**
Changed the way detectors are loaded

### CODE REVIEW

File: applications/smart-distancing/README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Smart Distancing is an open-source application to quantify social distancing mea
 
 You can run this application on edge devices such as NVIDIA's Jetson Nano or Google's Coral Edge-TPU. This application measures social distancing rates and gives proper notifications each time someone ignores social distancing rules. By generating and analyzing data, this solution outputs statistics about high-traffic areas that are at high risk of exposure to COVID-19 or any other contagious virus. The project is under substantial active development; you can find our roadmap at https://github.com/neuralet/neuralet/projects/1.
 
+We encourage the community to join us in building a practical solution to keep people safe while allowing them to get back to their jobs. You can read more about the project motivation and roadmap here: https://docs.google.com/presentation/d/13EEt4JfdkYSqpPLpotx9taBHpNW6WtfXo2SfwFU_aQ0/edit?usp=sharing
```
Comment (JsonSadler) [inline applications/smart-distancing/README.md:15]: This is actually git, not me.

File: applications/smart-distancing/smart_distancing/core/_cv_distancing.py [hunk 2]
```diff
@@ -26,18 +26,35 @@ def __init__(self, config):
         self.frame_counter = itertools.count(0)
         self.tracker = CentroidTracker(
             max_disappeared=int(self.config.get_section_dict("PostProcessor")["MaxTrackFrame"]))
+
+        detector_name_tmp = self.config.get_section_dict("Detector")["Name"]
         if self.device == 'Jetson':
-            from smart_distancing.detectors.jetson import MobilenetSsdDetector
-            self.detector = MobilenetSsdDetector(self.config)
+            if detector_name_tmp == 'mobilenet_ssd_v2_coco':
```
Comment (JsonSadler) [inline applications/smart-distancing/smart_distancing/core/_cv_distancing.py:32]: These should be listed somewhere, I still have inner issues with derivations of BaseDetector class and the role of "Name" in the detector's config.

**Review by JsonSadler — COMMENTED** [review]
Changed the way detectors are loaded.

File: applications/smart-distancing/smart_distancing/detectors/edgetpu/_detectors.py [hunk 3]
```diff
@@ -109,9 +109,11 @@ class MobilenetSsdDetector(EdgeTpuDetector):
 
 class PedestrianSsdDetector(EdgeTpuDetector):
     __doc__ = TFLITE_CLASS_DOC
+    DEFAULT_MODEL_URL = 'https://github.com/neuralet/neuralet-models/raw/master/edge-tpu/pedestrian_ssd_mobilenet_v2/'
```
Comment (JsonSadler) [inline applications/smart-distancing/smart_distancing/detectors/edgetpu/_detectors.py:112]: I also fixed these addresses.

**Review by mhejrati — APPROVED** [review]

**Pull request merged**